tasks/main.yml

```diff
@@ -1,2 +1,172 @@
 ---
-# tasks file for pacifica
+- name: Include Package Vars.
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+    - default.yaml
+  tags:
+    - pacifica
+    - pacifica_python
+
+- name: Include CherryPy Config.
+  include_vars:
+    file: defaults/cp_config.yml
+    name: default_cp_config
+  tags:
+    - pacifica
+    - pacifica_python
+
+- name: Install Setup Packages
+  package:
+    name: "{{ setup_packages }}"
+    state: present
+  async: "{{ pacifica_package_timeout }}"
+  tags:
+    - pacifica
+    - pacifica_python
+
+- name: Install Python Packages
+  package:
+    name: "{{ python_packages }}"
+    state: present
+  async: "{{ pacifica_package_timeout }}"
```

Comment:
Add ability to not install python by these means if desired in some cases:
```
+  when: external_python is undefined
```

Reply:
As discussed, as long as this package list is overridable when specifying an empty list in playbook vars someplace this should become superfluous. Noting here so we can know to validate current variable precedence in the ansible stack and how current location affects overriding methods.

Reply:
Done